scripts/release-afterdoc.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+main() {
+  local -r RELNAME="$1"
+  local -r RELEASE="$(curl -sSL -H "Accept: application/vnd.github.v3+json" "https://api.github.com/repos/kube-rs/kube-rs/releases/tags/${RELNAME}")"
+  local -r HURL="$(echo "${RELEASE}" | jq '.html_url' -r)"
+  # Skipping New Contributors highight from CHANGELOG + across repos for brevity and to avoid pinging them excessively
+  local -r BODY="$(echo "${RELEASE}" | jq '.body' -r | sd "## New Contributors[\w\W]*$" "")"
+  if grep -E "^${RELNAME} / " CHANGELOG.md; then
+    # We only run the script if the headline is unchanged (done at the end)
+
+    # Add in the body first
+    sd "(^${RELNAME} / [\d-]+\n===================\n)" "\$1${BODY}" CHANGELOG.md
+    # fix newlines caused by sd (^M)
+    sed -e "s/\r//g" CHANGELOG.md > CHANGELOG.m.md
+    mv CHANGELOG.m.md CHANGELOG.md
```

Comment:
Any reason to not just use `sed` directly?

Reply:
i have in general been trying to shift onto `sd` as the more modern tool (most replace calls in scripts/ use it), but uncovering some edge cases here and there... the first `sd` injects non-unix line endings.. not sure if it comes through from the jq or not, and only found this `sed` thing to fix it.

Reply:
Ah, looks like we already depend on it for the other scripts. I'm a bit wary of having undeclared dependencies on these less common tools, but I guess this isn't a new issue then.

Reply:
i've declared these in https://github.com/kube-rs/.github/blob/main/TOOLS.md which will be included on the website

Reply:
But again, I wouldn't consider this a blocker here since we already do depend on `sd` elsewhere.

Reply:
Maybe we'll just have to move to [`xtask`](https://github.com/matklad/cargo-xtask/) at some point...